ci-operator/config/openshift/odo/openshift-odo-master__v4.6.yaml

```diff
@@ -25,44 +25,6 @@ tag_specification:
   name: "4.6"
   namespace: ocp
 tests:
-- artifact_dir: /tmp/artifacts
-  as: integration-e2e
-  steps:
-    cluster_profile: aws
-    test:
-    - as: integration-e2e-steps
-      commands: scripts/openshiftci-presubmit-all-tests.sh
-      credentials:
-      - mount_path: /usr/local/ci-secrets/odo-rabbitmq
-        name: odo-rabbitmq
-        namespace: test-credentials
-      env:
-      - default: /usr/local/ci-secrets/odo-rabbitmq/amqpuri
-        name: ODO_RABBITMQ_AMQP_URL
-      from: oc-bin-image
-      resources:
-        requests:
-          cpu: "2"
-          memory: 6Gi
-    workflow: ipi-aws
-- artifact_dir: /tmp/artifacts
-  as: e2e-4x-psi
-  steps:
-    test:
-    - as: e2e-4x-psi-steps
-      commands: sh scripts/openshiftci-e2e-4x-psi-tests.sh
-      credentials:
-      - mount_path: /usr/local/ci-secrets/odo-rabbitmq
-        name: odo-rabbitmq
-        namespace: test-credentials
-      env:
-      - default: /usr/local/ci-secrets/odo-rabbitmq/amqpuri
-        name: ODO_RABBITMQ_AMQP_URL
-      from: oc-bin-image
-      resources:
-        requests:
-          cpu: "2"
-          memory: 6Gi
```

Comment:
@prietyc123  why are we moving PSI test related yaml to 4.7?

Reply:
It is of no reason but just because I thought to maintain only the periodic job configuration for the previous build other than latest.

Reply:
It will be hard to track then coz my [PR](https://github.com/openshift/release/pull/16212) adds the configuration at 4.6.
And it will become an extra effort if we start moving all the psi test to newer config file.
I think we should maintain only one file for PSI, i.e. `openshift-odo-master__v4.6.yaml`, then it will be easy to track.

Reply:
> it will become an extra effort if we start moving all the psi test to newer config file.
I think we should maintain only one file for PSI, i.e. openshift-odo-master__v4.6.yaml

IMO it would a great idea if we maintain psi thing inside `openshift-odo-master.yaml` rather `openshift-odo-master__v4.x.yaml` . Anyway we have this in plan for further psi configuration. @mohammedzee1000 correct me if I am wrong.